src/test/java/test/dataprovider/issue2327/IssueTest.java

```diff
@@ -0,0 +1,37 @@
+package test.dataprovider.issue2327;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.testng.TestListenerAdapter;
+import org.testng.TestNG;
+import org.testng.annotations.Test;
+import test.SimpleBaseTest;
+
+public class IssueTest extends SimpleBaseTest {
+
+  @Test(description = "GITHUB-2327")
+  public void runTest() {
+    TestNG testng = create(SampleTestCase.class);
+    TestListenerAdapter tla = new TestListenerAdapter();
+    testng.addListener(tla);
+    testng.run();
+
+    System.out.println("2327p:" + tla.getPassedTests().size());
```

Comment:
Please remove these sys out statements.

Reply:
It's out now. Sorry for that. Was just to debug some things on travis.
Test should be green now.